home/modules/contribute/pages/pages.adoc

```diff
@@ -1,75 +1,124 @@
 
-The title of a document is placed on the first line of the file.
-It is specified by one equals sign (`=`), followed by one blank space, and then the text of the title.
-*All AsciiDoc documents stored in a _pages_ directory must have a document title*.
+Put the title of your AsciiDoc document on the first line of the file. 
+Mark the title with one equals sign (`=`), followed by a blank space, and then the title text. 
 
-On the line directly below the title and subsequent contiguous lines, you can add xref:attributes-and-roles.adoc[document attributes].
-Each attribute must be entered on its own line.
-Document attributes are optional.
+NOTE: Every AsciiDoc file stored in a *pages* directory must have a document title.
+
+On the second line and any following lines, you can add xref:attributes-and-roles.adoc[document attributes].
+Each attribute must be entered on its own line, as shown:
 
 ----
 = The Title of My Document
-:page-type: concept
+:page-topic-type: concept
+:description: This is the description of my page! It's short and sweet.
 :url-query-tut: https://query-tutorial.couchbase.com/tutorial/#1
 
+[abstract]
+{description}
+
 This is the body of my document.
 Let's get started with N1QL!
 ----
 
-A single blank line signals the end of the header.
-The document body starts on next line that has text.
+You must include the following document attributes for every AsciiDoc file stored in a *pages* directory: 
 
-== Document Sections
+include::partial$available-attributes.adoc[tag=required]
 
-A document is divided into sections by heading titles.
-Heading 2 section titles, indicated by two equals signs (`==`), are displayed in the xref:nav-menus-and-files.adoc[On This Page sidebar].
+A single blank line signals the end of the header.
+The <<doc-body,document body>> starts on the next line that has text.
 
-----
-== Heading 2 Section Title (H2)
+You can include your own custom document attributes, or make use of some of these other useful attributes, based on your page's content: 
 
-=== Heading 3 Section Title (H3)
+include::partial$available-attributes.adoc[tag=optional]
 
-==== Heading 4 Section Title (H4)
-----
+[#doc-body]
+=== Document Body
+
+Write your document body using appropriate xref:basics.adoc[AsciiDoc syntax]. 
 
-*Don't nest sections out of order*.
-For example, a heading 4 section can't be directly inside a heading 2 section; it must be nested inside a heading 3 section.
+Consider making use of: 
 
-*Don't apply bold formatting to headings*.
+* {basics-link}#bold-and-code-styles[Bold and Code Styles]
+* {basics-link}#ui-macros[Button, Keyboard, and Menu UI Macros]
+* {basics-link}#urls[URLs]
+* {basics-link}#description-lists[Description Lists]
+* {basics-link}#ordered-lists[Ordered Lists]
+* {basics-link}#unordered-lists[Unordered Lists]
+* {basics-link}#images[Images]
+* {basics-link}#admonitions[Admonitions]
+* {basics-link}#tables[Tables]
 
 [#ventilate]
-== Ventilated Prose (aka Sentence Per Line)
```

Comment:
not a big issue, but I kinda like the "immediately understandable summary" in title